generator/src/html/script/Main.hx

```diff
@@ -19,34 +18,27 @@ class Main {
 		// parse and locate myselft
 		assert(tocData != null, "toc data is missing");
 		var toc = J(JQuery.parseHTML(tocData));
-        
-        var baseURI = null;// document.baseURI ;
-        //FIXME IE workaround for the lack of document.baseURI 
-        if (baseURI == null) {
-            if (document.getElementsByTagName("base").length > 0){
-                var splitdocpath = document.URL.split("/");
-                js.Browser.console.log ("pop = " + splitdocpath.pop());
```

Comment:
err, you accidentally removed this...

so... broken still: https://brtguide.itdp.org/pull_request/117/guide/volume/business-plan.html

will revert